### PR TITLE
[Nicole] add Don't Fly Too High and it's German counterpart

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: database/N/Nicole/Don’t Fly Too High/Don’t Fly Too High [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+______________________________________________________________________________
+Language                 English
+Name                     Don’t Fly Too High
+Artist                   Nicole
+Album                    Don’t Fly Too High
+Year                     1981
+MusicBrainz ID           9cf590fc-5697-4a0b-9381-be1588b15465
+Original text by         Jean Frankfurter, Robert Jung, John Möring
```
Comment (snshn) [inline database/N/Nicole/Don’t Fly Too High/Don’t Fly Too High:52]:
```suggestion
Original text by         Jean Frankfurter,  Robert Jung,  John Möring
```

File: database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+______________________________________________________________________________
+Language                 German
+Name                     Flieg nicht so hoch, mein kleiner Freund
+Artist                   Nicole
+Album                    Flieg nicht so hoch, mein kleiner Freund
+Track no                 1
+Year                     1981
+MusicBrainz ID           499a2915-f870-4cb8-8b5e-9c861c22674b
+Original text copyright  Jean Frankfurter, Robert Jung
```
Comment (snshn) [inline database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund:49]:
```suggestion
Original text copyright  Jean Frankfurter,  Robert Jung
```

File: database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Flieg nicht so hoch mein kleiner Freund
+Die Sonne brennt dort oben heiß
+Wer zu hoch hinaus will der ist in Gefahr
+Flieg nicht so hoch mein kleiner Freund
+Glaub' mir ich mein es gut mit dir
+Keiner hilft dir dann, ich weiß es ja
+Wie's damals bei mir war
+
+
+______________________________________________________________________________
```
Comment (snshn) [inline database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund:41]:
```suggestion
_______________________________
```

File: database/N/Nicole/Don’t Fly Too High/Don’t Fly Too High [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+'Cause if you try you may get hurt
+You will burn your wings
+When you reach for the sun
+Don't fly too high, my little bird
+Your lonely cry won't be heard
+You'll be tumbling down to the ground
+'Cause pride will have a fall
+
+
+______________________________________________________________________________
```
Comment (snshn) [inline database/N/Nicole/Don’t Fly Too High/Don’t Fly Too High:45]:
```suggestion
________________________________
```

**Review by snshn — COMMENTED** [review]
Double spaces are needed to make it possible to parse it as an array (e.g. `Company,*LLC,**Other Company,*LLC`)

File: database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund (hunk 3 above)
Comment (andy5995) [reply database/N/Nicole/Flieg nicht so hoch, mein kleiner Freund/Flieg nicht so hoch, mein kleiner Freund:41]: What's wrong here @snshn ? The line is too long?

**Review by snshn — APPROVED** [review]

**Pull request merged**